icclim/generic_indices/generic_indicators.py

```diff
@@ -1149,3 +1135,9 @@ def _check_data(climate_vars: list[ClimateVariable], src_freq: str):
         da = climate_var.studied_data
         if "time" in da.coords and da.time.ndim == 1 and len(da.time) > 3:
             check_freq(da, src_freq, strict=True)
+
+
+def _is_rate(query: Quantity | DataArray) -> bool:
```

Comment:
User xclim unit_registry `Quantity` for consistency (even though for typing it has no effect).

Reply:
Actually, it's better to use pint.Quantity in typing. It seems the actual type is not always properly resolved when using a type from a runtime created unit registry. At least with pycharm.